app/code/Magento/Catalog/Model/Category.php

```diff
@@ -87,7 +87,7 @@ class Category extends \Magento\Catalog\Model\AbstractModel implements
      *
      * @var string
```

Comment:
are we sure that this is a string? 
:robot: the init value is **false**

Reply:
Hi @rvitaliy, please ignore any PR with label "https://github.com/magento/magento2/labels/Catalog%20Storefront"